fe/fe-core/src/main/java/org/apache/doris/metric/MetricRepo.java

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.doris.metric;
 
+import java.util.ArrayList;
```

Comment:
Thanks for your PR.
Please reorganize the import order, you can refer to
http://doris.incubator.apache.org/master/zh-CN/developer-guide/java-format-code.html#import-order